cypress/api/task1.test.js

```diff
@@ -0,0 +1,49 @@
+describe('ToDo Lists', () => {
+  const baseUrl = "http://localhost:3030/"   
+  beforeEach(() => {
+    cy.visit(`${baseUrl}`)
+    cy.get('div#root button[type="button"]:nth-child(3)').click()
+  })
+  it('retrieves lists', () => {
+    cy.request(`${baseUrl}/lists`)
+      .then(response => {
+        expect(response.status).to.eq(200)
+      })
+  })
+  it('invalid endpoint for get request', () => {
+    cy.request({
+      url: `${baseUrl}/invalid`, 
+      failOnStatusCode: false 
+    })
+      .then(response => {
+        expect(response.status).to.eq(404)
+      })
+  });
+  it('add a list', () => {
+    cy.request({
+      method: 'POST', 
+      url: `${baseUrl}/lists/add`, 
+      body: {
```

Comment:
I would create a variable called `const list = { name: 'School', items: [] }` at the beginning of the test and write the API call in the following way:
```cy.request('POST', '${baseUrl}/lists/add', list)
It makes the test much easier to read.